resources/language/resource.language.es_es/strings.po

```diff
@@ -670,7 +673,7 @@ msgstr "Nuevos lanzamientos"
 
 msgctxt "#30171"
 msgid "Trending now"
-msgstr "Tendencias ahora"
+msgstr "Tendencia ahora"
```

Comment:
DON'T AGREE, because I think there are several trends and not just one.

Reply:
AGREE